app/components/UI/ManageNetworks/ManageNetworks.test.js

```diff
@@ -23,15 +23,12 @@ jest.mock('react-redux', () => ({
   useSelector: jest.fn(),
 }));
 
-const mockProviderConfig = {
-  type: 'mainnet',
-  chainId: '1',
-};
+const mockProviderConfig = 'Ethereum Main Network';
```

Comment:
This name doesn't seem accurate, I think what we want it's more
```suggestion
const mockNetworkName = 'Ethereum Main Network';
```